lib/europeana/api/client.rb

```diff
@@ -34,8 +34,8 @@ def connection
             conn.request :parameter_repetition
             conn.request :authenticated_request
             conn.request :retry, max: 5, interval: 3,
-                                 exceptions: [Errno::ECONNREFUSED, Errno::ETIMEDOUT, 'Timeout::Error',
-                                              Faraday::Error::TimeoutError, EOFError]
+                                 exceptions: [Errno::ECONNREFUSED, Errno::ETIMEDOUT, 'Timeout::Error', EOFError]
```

Comment:
I think we will want to also stop retrying on the other timeout exception classes, i.e. `Errno::ETIMEDOUT` and `Timeout::Error`.